index.bs

```diff
@@ -746,12 +746,36 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
     1. If {{AuthenticatorSelectionCriteria/authenticatorAttachment}} is [=present|present=] and its value is not equal
         to |authenticator|'s attachment modality, [=iteration/continue=].
-    1. If {{AuthenticatorSelectionCriteria/requireResidentKey}} is set to |true| and the |authenticator|
+    1. If {{AuthenticatorSelectionCriteria/requireResidentKey}} is set to `true` and the |authenticator|
         is not capable of storing a [=Client-Side-Resident Credential Private Key=], [=iteration/continue=].
-    1. If {{AuthenticatorSelectionCriteria/requireUserVerification}} is set to |true| and the
+    1. If {{AuthenticatorSelectionCriteria/requireUserVerification}} is set to {{UserVerificationRequirement/required}} and the
         |authenticator| is not capable of performing [=user verification=], [=iteration/continue=].
 
-    1. Let |excludeCredentialDescriptorList| be a new [=list=].
+    1. Let |userVerification| be the <dfn>effective user verification requirement for credential creation</dfn>, a Boolean value,
+        as follows. If {{AuthenticatorSelectionCriteria/requireUserVerification}}
+
+            <dl class="switch">
```

Comment:
Actually, this procedure is not correct for creating a credential. Once a device has been set with UV option or ClientPin semantics, authenticator always asks for user verification while creating a credential. So there is no option to skip it. Skipping is available for authenticatorGetAssertion. 

So for creating a credential, from the web, it has to be a Boolean and not an enum.  Then if it `true` (meaning RP wants user verification), Platform can reject the call if authenticator does not support it via any of the authenticator's mechanism. In case of ClientPin semantics, if authenticator supports a PIN but it has not been set up yet, platform can first provision or set up a PIN and then try for authenticatorMakeCredential. 

If it is set to `false`, and authenticator is configured with ClientPin or in built `UV` mechanism, platform can reject the call as authenticator always requires user verification while creating a credential. For example, if ClientPin is setup for the authenticator, you have to provide PIN for creating a credential.

Reply:
Good points. I don't agree that the platform should necessarily reject authenticators that require UV when the RP doesn't, but I've made the other changes. I left the PIN setup and entry unspecified as before.